test/unit/alignment/configuration/align_config_gap_cost_affine_test.cpp

```diff
@@ -0,0 +1,54 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alignment/configuration/align_config_gap_cost_affine.hpp>
+#include <seqan3/core/algorithm/configuration.hpp>
+#include <seqan3/std/concepts>
+
```

Comment:
Travis throws errors because it doesn't know the get function I think. So this should fix it I hope. ( https://docs.seqan.de/seqan/3-master-user/group__algorithm.html#ga8f02011be28ea1c73ca39a46fd5ce613 )
```suggestion
 using seqan3::get; 
```